INCode/diagramconfiguration.py

```diff
@@ -12,52 +13,79 @@ class TreeColumns(IntEnum):
 
 
 class CallableTreeItem(QTreeWidgetItem):
-    def __init__(self, callable, index, parent=None):
+    def __init__(self, callable, parent=None):
         super(QTreeWidgetItem, self).__init__(parent)
-        self.index_ = index
+
+        self.common_path_ = ''
+        if isinstance(parent, CallableTreeItem):
+            parent.referenced_items_.append(self)
+            self.common_path_ = parent.common_path_
```

Comment:
Consider to factor the common path out to a central location. Related to https://github.com/rknuus/INCode/pull/1/files#r251864190